lib/fhir_models/bootstrap/generator.rb

```diff
@@ -196,10 +204,11 @@ def generate_class(hierarchy, structure_def, top_level = false)
                 field.binding.delete('extension')
                 # set the actual code list
                 binding_uri = field.binding['uri']
-                binding_uri = binding_uri[0..-7] if binding_uri&.end_with?('|4.0.0')
+                # Strip off the |4.0.0 or |4.0.1 or |2014-03-26 or similar from the ends of URLs
+                binding_uri&.gsub!(/\|[A-Za-z0-9\.\-]*/, '')
                 codes = @defn.get_codes(binding_uri)
                 field.valid_codes = codes unless codes.nil?
-                if field.valid_codes.empty? && binding_uri && !binding_uri.end_with?('bcp47', 'bcp13.txt', 'mimetypes')
+                if field.valid_codes.empty? && binding_uri && !binding_uri.end_with?('bcp47', 'bcp13.txt', 'mimetypes', 'LL379-9')
```

Comment:
It might be nice to move this list of missing expansion uri endings to the top of the file into its own variable so it clear that this is happening.  Is there really binding_uri that end with `bcp13.txt`?

Reply:
> Is there really binding_uri that end with bcp13.txt?

Unfortunately, yes.

Reply:
Moving into constant.